### PR TITLE
README reworked and Weblate mentioned

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,24 +33,26 @@ You can find the FAQ here (same as in app): https://ics-openvpn.blinkt.de/FAQ.ht
...
+The AIDL API offers real control. (More about this in the developing section.) \
+Due to high demand also acitvies to start/stop, pause/resume (like a user would with the notification) exists:
   
  - `de.blinkt.openvpn.api.DisconnectVPN`
  - `de.blinkt.openvpn.api.ConnectVPN`
  - `de.blinkt.openvpn.api.PauseVPN`
  - `de.blinkt.openvpn.api.ResumeVPN`
 
-They use `de.blinkt.openvpn.api.profileName` as extra for the name of the VPN profile..
+They use `de.blinkt.openvpn.api.profileName` as extra addition for the name of the VPN profile…
```
Comment (comradekingu) [inline README.md:44]: ?

**Comment by schwabe** [comment]
no. As extra is correct here as it is used as noun. Please do not try to correct technical documentation you don't understand.

**Comment by comradekingu** [comment]
@schwabe I'll happily change it back. Just looks odd without "an extra" or "extra __".
The main idea here is to mention Hosted Weblate, as it is required to use their libre service/plan.

File: README.md [hunk 2]
```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
...
 <a href="https://f-droid.org/repository/browse/?fdid=de.blinkt.openvpn" target="_blank">
 <img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=de.blinkt.openvpn" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
 Developing
----------------
-If you want to develop on ics-openvpn please read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or emailing me. 
+----------
+Read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or e-mailing.
```
Comment (schwabe) [inline README.md:18]: Completely changes the meaning of this sentence.

File: README.md [hunk 3]
```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
...
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
 Developing
----------------
-If you want to develop on ics-openvpn please read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or emailing me. 
+----------
+Read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or e-mailing. 
 
-Also please note that before contributing to the project that I would like to retain my ability to relicense the project for different third parties and therefore probably need a contributer's agreement from any contributing party. To get started, [sign the Contributor License Agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn).
+To retain my ability to re-license the project for different third-parties I probably need a [contributor agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn) from any contributing party.
```
Comment (schwabe) [inline README.md:20]: fixes a mistake but also changes the tone and removes an explaination.

File: README.md [hunk 4]
```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
...
 
-You can help
+Translations
 ------------
-Even if you are no programmer you can help by translating the OpenVPN client into your native language. [Crowdin provides a free service for non commercial open source projects](https://crowdin.net/project/ics-openvpn/invite) (Fixing/completing existing translations is very welcome as well)
+Help [translate](https://hosted.weblate.org/projects/openvpn-for-android/) the app to your language.
+<a href="https://hosted.weblate.org/engage/openvpn-for-android/">
+<img src="https://hosted.weblate.org/widgets/openvpn-for-android/-/horizontal-blue.svg" alt="Oversettelsesstatus" />
+</a>
```
Comment (schwabe) [inline README.md:27]: You just assumed that I will switch to weblate. I haven't decided that yet. I haven't found the time to actually look into weblate and if it is really that much better as you claim, so don't make decisions for me.

File: README.md [hunk 5]
```diff
@@ -59,11 +65,11 @@ For example `ca mycafile.pem` becomes
 ```
 Footnotes
 -----------
-Please note that OpenVPN used by this project is under GPLv2. 
+Please note that OpenVPN used by this project is under GPLv2+.
 
-If you cannot or do not want to use the Play Store you can [download the apk files directly](http://plai.de/android/).
+If you cannot or do not want to use the Play Store you can [download the APK files directly](http://plai.de/android/).
```
Comment (schwabe) [inline README.md:70]: apk is written in lowercase, https://en.wikipedia.org/wiki/Apk_(file_format)

File: README.md [hunk 6]
```diff
@@ -59,11 +65,11 @@ For example `ca mycafile.pem` becomes
...
 
-If you want to donate you can donate to [arne-paypal@rfc2549.org via paypal](https://www.paypal.com/cgi-bin/webscr?hosted_button_id=R2M6ZP9AF25LS&cmd=_s-xclick), or alternatively if you believe in fancy Internet money you can use Bitcoin: 1EVWVqpVQFhoFE6gKaqSkfvSNdmLAjcQ9z 
+If you want to be a hero, donate to [arne-paypal@rfc2549.org via paypal](https://www.paypal.com/cgi-bin/webscr?hosted_button_id=R2M6ZP9AF25LS&cmd=_s-xclick), or alternatively if you believe in fancy Internet money (or not) you can use Bitcoin: 1EVWVqpVQFhoFE6gKaqSkfvSNdmLAjcQ9z
```
Comment (schwabe) [inline README.md:72]: again, you change the tone/meaning here.

**Comment by schwabe** [comment]
@comradekingu in your changes you assume in a lot of things that you know better than me what I want to express with my stuff. That is something that just does not fly with me.

File: README.md (hunk 4 above)
Comment (comradekingu) [reply README.md:27]: This is one of the requirements to use it gratis though. It is your choice to merge.

File: README.md (hunk 5 above)
Comment (comradekingu) [reply README.md:70]: That is a mistake on Wikipedia. The rest of the article has it right.
Possibly it was meant to be ".apk" in the heading. (Which is a possible choice here too.)
Also, https://developer.android.com/guide/components/fundamentals

File: README.md (hunk 6 above)
Comment (comradekingu) [reply README.md:72]: I want the most amount of money coming your way.
The psychology of asking for donations stems from perceived value insofar as it is communicated, not the action itself. Repeating the word "donate" this way doesn't do anything other than focus on the ask.
Secondarily, people like to show virtue by visibly donating.
There is a lot more I would do, but I tried to make minimal changes to your text.

Liberapay is a good platform to accept donations btw. Only transactional cost from payment processors, and easier to set up than Collectives.
(I noticed it should be "PayPal" just now".)

File: README.md (hunk 3 above)
Comment (comradekingu) [reply README.md:20]: I introduced a mistake too in forgetting "contributor **license** agreement".
Can "probably" be removed? It reads like "pending research" to me.
The tone I interpret as defensive, as opposed to welcoming changes.
"The project" is repeated. I don't know what is lost other than length from a long-winded way to say (I thought) the same thing.

File: README.md (hunk 2 above)
Comment (comradekingu) [reply README.md:18]: It is implied the user can e-mail about things other than things related to issues, but that doesn't need explaining.
With the *before*, the original reads ~"do you really have to do this", and ~"it is likely to annoy" to me.

File: README.md (hunk 6 above)
Comment (schwabe) [reply README.md:72]: You start to really annoy me with assuming what would be best for me. Have you ever thought that I don't want to entice people to donate and just this neutral tone is exactly what I want?

Comment (comradekingu) [reply README.md:72]: Never crossed my mind. I only know what is on the table or communicated otherwise.

**Pull request closed without merging**